lib/handler/status/durations.c

```diff
@@ -125,12 +134,23 @@ static h2o_iovec_t durations_status_final(void *priv, h2o_globalconf_t *gconf, h
             "request-total-time") "," DURATION_FMT("process-time") "," DURATION_FMT("response-time") "," DURATION_FMT("duration"),
         DURATION_VALS(connect_time), DURATION_VALS(header_time), DURATION_VALS(body_time), DURATION_VALS(request_total_time),
         DURATION_VALS(process_time), DURATION_VALS(response_time), DURATION_VALS(total_time));
-
-#undef BUFSIZE
 #undef DURATION_FMT
 #undef DURATION_VALS
-
+    char *delim = "";
+    ret.len += sprintf(ret.base + ret.len, ",\n\"evloop-latency-nanosec\": [");
+    for(int i = 0; i < agg_stats->stats.evloop_latency_nanosec.size; i++) {
+        size_t len = snprintf(NULL, 0, "%s%llu", delim, agg_stats->stats.evloop_latency_nanosec.entries[i]);
+        /* require that there's enough space for the closing "]\0" */
+        if (ret.len + len + 1 >= BUFSIZE)
+            break;
+        ret.len += snprintf(ret.base + ret.len, BUFSIZE - ret.len, "%s%llu",
+                            delim, agg_stats->stats.evloop_latency_nanosec.entries[i]);
+        delim = ",";
+    }
+    ret.len += snprintf(ret.base + ret.len, BUFSIZE - ret.len, "]");
+#undef BUFSIZE
     duration_stats_free(&agg_stats->stats);
+    free(agg_stats->stats.evloop_latency_nanosec.entries);
```

Comment:
I think we'd want to move this into `duration_stats_free`. It's part of `st_duration_stats_t`, and initialized by `duration_status_init` (I think the functions should be named `duration_stats_init` and `duration_stats_dispose`, but that's another issue).